### PR TITLE
Allow submitting jobs via krel stage/release

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
This change allows us to directly use `krel stage/release` for submitting jobs to GCB.
#### Which issue(s) this PR fixes:

Refers to #1673 
##### :heavy_check_mark: Stage:
```
> export TOOL_ORG=saschagrunert && \
  export TOOL_BRANCH=krel-submit && \
  go run ./cmd/krel stage --type beta
```
https://console.cloud.google.com/cloud-build/builds/4f32ded1-a13f-469d-a1a9-b5555ceb83f4?project=kubernetes-release-test

##### :heavy_check_mark: Release:
```
> export TOOL_ORG=saschagrunert && \
  export TOOL_BRANCH=krel-submit && \
  go run ./cmd/krel release --type official --branch release-1.19 --build-version v1.19.4-rc.0.51+5f1e5cafd33a88
```
https://console.cloud.google.com/cloud-build/builds/885226c9-4d5f-4e1f-b782-ecb967cb1125?project=kubernetes-release-test
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by justaugustus** [comment]
Very cool!
/lgtm
/approve

**Review by hasheddan — APPROVED** [review]
/lgtm

**Comment by saschagrunert** [comment]
I think I have to rebase this one after the other PRs got merged
/hold

**Comment by saschagrunert** [comment]
Rebased.
/hold cancel

**Pull request merged**
